pacemaker.spec.in

```diff
@@ -403,14 +403,26 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun_with_restart pacemaker.service
 
+%pre remote
+systemctl --quiet is-active pacemaker_remote
+if [ $? -eq 0 ] ; then
+    touch %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
```

Comment:
Question is if and when to remove the subdir which is why I was more in
favour for the dot-aproach. But if Fedora recommends it ...
Found a tangling gconf subdir on my machine so I'll leave the subdir as
well - should be the safest when used by subpackets and scriptlets in parallel.